web/utils/types/post.ts

```diff
@@ -267,11 +267,13 @@ export const trainedLabels: Record<Trained, string> = {
 export enum Status {
   Yes = "yes",
   No = "no",
+  Unknown = "unknown",
```

Comment:
The reason I added the unknown field was because it seems like spayNeuterStatus uses the Status enum and we need an unknown value to filter for unknown spay/neuter statuses as well, should I change spayNeuterStatus to use the Trained enum?

Reply:
Ah, you are right. Spay/neuter status is yes/no/unknown on the frontend. For posts and related, this needs to be updated to allow yes/no/unknown like the `Trained` enum. I'm thinking we rename the enums to be more descriptive of the options themselves instead of what they stand for (like `YesNo` and `YesNoUnknown`). The spay/neuter status needs to stay the same for User, though, so watch out for that.

Reply:
Good catch. When a user selects "Spayed/Neutered" in the feed filters, we want to only show posts that have `spayNeuterStatus: "yes"`. Since the backend is taking in an _optional_ input of `spayNeuterStatus`, this field should only be included in the query input if the option is checked.

Therefore, we do need to update line 185 of `routers/post.ts` to make the else value of the ternary to be `Object.values(Trained)` instead. The input can remain dependent on `Status` given that we only send this field if the frontend has the checkbox selected.